packages/react-jsx-highcharts/src/components/PlotBandLine/index.js

```diff
@@ -1,6 +1,7 @@
 import PlotBand from './PlotBand';
 import PlotLine from './PlotLine';
 import PlotBandLineLabel from './PlotBandLineLabel';
+import usePlotBandLine from './usePlotBandLine';
```

Comment:
As already stated by Travis, looks like this file hasn't been added?

Reply:
Sorry for that. Added